### PR TITLE
Group pr 2022 06 18

### PR DESCRIPTION
**Pull request by Volvo-hackangel**
needs a linbreak check

### CODE REVIEW

File: script/Arcueid/Day 11/11_00_ARC11_2.txt [hunk 1]
```diff
@@ -929,7 +929,7 @@ A final paradise, sprawled among the mountains.
 [sha:62e43bd98536e70dc341b01ba02c5fe8e4b2b461]{
 -- Page 99, Offset 29779.
 -- 星の内海を思わせる黎明の原野。
-A dawn wilderness that calls to mind a cove of stars.
+A dawn wilderness that calls to mind the %{i}Inner-sea of the Planet%{/i}.
```
Comment (rschlaikjer) [inline script/Arcueid/Day 11/11_00_ARC11_2.txt:932]: Dupe comment from other PR, but no hypen

File: script/Arcueid/Day 13/13_00_ARC13_5.txt [hunk 2]
```diff
@@ -63,7 +63,7 @@ A fraction of those people, such as you and I, come to possess special abilities
 [sha:492dda4d771735e11b44feffcc54e1dc017eec71]{
 -- Page 408, Offset 32831.
 -- 私はこの極点に到達するまで十六回もの死を体験してきたが―――そちらはただの一度だけか。
-I had to experience death firsthand sixteen times before I finally reached that summit―――yet you have experienced it only once.
+I had to experience death sixteen times before I could arrive at this Focality―――yet you did so by only experiencing it once.
```
Comment (rschlaikjer) [inline script/Arcueid/Day 13/13_00_ARC13_5.txt:66]: Might keep `finally` to encode the 十六**も**, also prefer `after experiencing`
> ...sixteen times before I finally arrived at this Focality―――yet you did so after only experiencing it once.

File: script/Ciel/Day 5/05_01_CIEL05_4.txt [hunk 3]
```diff
@@ -862,7 +862,7 @@ Right now, I can completely and utterly kill you.
 -- Page 728, Offset 12823.
 -- 星の内海、ソラを覆う天蓋は謳う。
 // not ruby in jp, but sky might possibly be changed or rubied with cosmos, since sora is in katakana
-The inner sea of the planet, the canopy covering the <sky|c o s m o s  > applauds.
+The Inner-sea of the Planet, the canopy covering the <sky|c o s m o s  > applauds.
```
Comment (rschlaikjer) [inline script/Ciel/Day 5/05_01_CIEL05_4.txt:865]: I wouldn't hyphenate

File: system_strings/sysmes_text.en [hunk 4]
```diff
@@ -635,13 +635,13 @@ Akiha-sama Returns
...
-A foolish brother ought to die.
-Insincere Suspicious Policemen
+Foolish brothers ought to die.
+Sleazy Policemen
 I'll ask what he wants.
 I'll ignore them.
 Slipping back into a routine.
 Do my utmost to intercept her.
-Do my utmost to focus on lesson.
+Try to focus on the lesson.
```
Comment (rschlaikjer) [inline system_strings/sysmes_text.en:644]: I'd keep the pattern here, the choices in game do IIRC

File: system_strings/sysmes_text.en [hunk 5]
```diff
@@ -1111,20 +1111,20 @@ More vampire murders have been^reported. Is this case not yet^over...?
...
+Her ability to make even adults^cower in fear is amazing. Though^she may seem calm, the family^head is still really immature.
+I feel a strange atmosphere^emanating from the living room.^Hisui appears, and stoically^informs me that I have a^visitor...
 I've made my desperate escape^from the mansion. I shudder to^think what's in store for me^later, but I need to talk to^this woman now.
 I question Arcueid about the^vampire incidents. The truth^is much too simple.
-Arcueid, the hunter of Undead.^I'm once again reminded of the^condition of the city, which has^been contaminated by the Dead...
-I question what I, as a human,^can do. However, there's no^getting out of it now, so I^offer to help Arcueid――
-We're starting our nighttime^investigation. I accept Arcueid's^proposal and head out into the^city without my glasses on.
-I'll talk to her a little more.^Now that I've decided to help^her exterminate vampires, there's^too much I still don't know.
+Arcueid hunts the Undead.^I am once again struck by the state^of the city, infected by^the Dead Apostles...
+I know a human shouldn't think like^this, but I can't help but offer to^cooperate with Arcueid.———
```
Comment (rschlaikjer) [inline system_strings/sysmes_text.en:1120]: What's with the `.―`

File: system_strings/sysmes_text.en [hunk 6]
```diff
@@ -1155,25 +1155,25 @@ There's no way I can talk my^way out of this. I'll go with^a full-speed, full-th
...
 This laid-back guy just started^talking to me. Is this guy^really a police officer...?
 I can't deal with them^effectively if I don't even^know what it is they're after.^I should find out more about^their intentions.
-There's too much I don't^understand about the events^of the past few days. I should^avoid engaging them unprepared^and get away from them...
+There's too much I don't^understand about the events^of the past few days. I should^get away from here instead of carelessly^engaging.
 I can't concentrate on the^lesson at all. I casually^glance out the window, and^am shocked to discover...
 I head to the courtyard at full^speed. I need to lure her away^before she gets the chance to^cause any mischief...!
 There's nothing I can do. I'll^concentrate on the lesson. I^need a miracle to happen right^now.
-I didn't see anything. I'll just^pretend to sleep and wait for it^to pass. Twenty minutes until the^class ends...!
-Once lunch break comes around,^Arihiko shows up. I don't think^there's anything unusual going^on in town, but I should at^least ask.
+I didn't see anything. I'll just^pretend to sleep and wait for it^to pass. Only twenty minutes^until the class ends...!
```
Comment (rschlaikjer) [inline system_strings/sysmes_text.en:1162]: I'd drop the `the` before class

File: system_strings/sysmes_text.en [hunk 7]
```diff
@@ -1155,25 +1155,25 @@ There's no way I can talk my^way out of this. I'll go with^a full-speed, full-th
...
+Once lunch break comes around,^Arihiko shows up.I^might as well ask him if^there hasn't been anything unusual^happening in town.
```
Comment (rschlaikjer) [inline system_strings/sysmes_text.en:1163]: Use of negative `hasn't` here does't sit right
> I might as well ask him if there's been anything unusual happening in town.

File: system_strings/sysmes_text.en [hunk 8]
```diff
@@ -1155,25 +1155,25 @@ There's no way I can talk my^way out of this. I'll go with^a full-speed, full-th
...
-I'm curious about what Arihiko^said, but I doubt it's related^to the current case. I'll head^back to the mansion and preserve^my strength.
+I'm curious about what Arihiko^was talking about, but I doubt it's^related to the current case.^I'll head back to the mansion and^preserve^my strength.
 I return to the mansion before^sunset. I wanted to talk to^Akiha, but she hasn't returned^home yet.
 There's no way I'd be able to^deceive Hisui. I'll at least^be upfront about me leaving^the house at night.
-There's no way I could tell her^the truth. I feel bad for Hisui,^but I'll have to deceive her.
-It's lights-out time. I'm^keeping Arcueid waiting. I^leave my room and head to^the park where we promised^to meet.
-A community of vampires. That^apartment complex was right in^the middle of the residential^area.
-We've returned to the park and^I'm having a rest. Arcueid tells^me about the Mystic Eyes of Death^Perception.
+There's no way I could tell her^the truth. I feel bad for doing^this to Hisui, but I'll have^to deceive her.
+It's past lights-out. I'm^keeping Arcueid waiting. I^leave my room and head to^the park where we promised^to meet.
```
Comment (rschlaikjer) [inline system_strings/sysmes_text.en:1169]: I'd use `after` rather than `past` here

File: system_strings/sysmes_text.en [hunk 9]
```diff
@@ -1189,14 +1189,14 @@ I feel her blatant animosity^and killing intent. I recognize^her as an enemy. I'
 アルクェイドもほっとけないが、^シエル先輩も大切なひとだ。^敵とは思いたくはない。
 新天地を求め集まった若者達。^振り降ろされた凶器は容赦なく、^彼らの体を圧し潰してゆく―――
 まるで寝付く事ができない。^今夜も本を手に取った。^読んでいれば眠気も来るだろう。
-Everything is so ridiculous I'm^unable to process it properly.^But if there's one thing that^sticks out, it's...
+Everything has been so crazy lately.^I'm unable to process it properly.^But if there's one thing that^sticks out, it's...
 I think back on Arcueid. I'm^worried about her expression^from last night. Have her wounds^not fully healed yet?
-I think back on the shadowy^figure. Was he targetting me?^Well, not like I'll be seeing^it again.
-I think back on Noel...sensei?^Is she a nun? Well, I guess she^is.
+I think back on the shadowy^figure. Was he targetting me?^Well, it's not like I'll be^seeing it again.
```
Comment (rschlaikjer) [inline system_strings/sysmes_text.en:1194]: Only one `t` in `targeting`

File: system_strings/sysmes_text.en [hunk 10]
```diff
@@ -1230,31 +1230,31 @@ I can't break my promise with^Arcueid. I have to somehow slip^behind their backs
...
-The Holy Church's Presbyter^speaks. About her 'enemy's'^true form, and his purpose.
+Test it out', she said. Her^breath approaches my nape. In^the face of this inexplicable^fear, I――
+Even though it was supposed to^be a joke, Arcueid apologizes^and leaps away.
+The Presbyter from the Holy Church^speaks about her enemy's^true form, and his purpose.
 I can't afford to waste any more^time. I rush into the nighttime^city to chase after Arcueid.
 I can't find Arcueid. If she was^here, she'd be――
-I'll go to her room. She might^have given up searching for the^Dead for tonight, and gone back^to her room.
-I don't think she went back^to her room. If she was to go^somewhere, it'd be somewhere^no one would go to―――
-I remove my glasses and head^into the nighttime city. She's^the only one whose death I can't^see. I'll believe that'll help^me find her.
+I'll go to her room. She might^have given up on searching for ^Dead tonight and gone back^to her room.
```
Comment (rschlaikjer) [inline system_strings/sysmes_text.en:1253]: space before ^, does that matter?

File: system_strings/sysmes_text.en [hunk 11]
```diff
@@ -1230,31 +1230,31 @@ I can't break my promise with^Arcueid. I have to somehow slip^behind their backs
...
+I don't think she went back^to her room. If she was to go^somewhere, it'd be a place^no one would go to―――
+I remove my glasses and head^into the night. She's^the only one whose death I can't^see. I'll believe that'll help^me find her.
```
Comment (rschlaikjer) [inline system_strings/sysmes_text.en:1255]: `I'll believe` is a bit weird, I think just `I believe`? Maybe I should port this file to the includes-JP format

File: system_strings/sysmes_text.en [hunk 12]
```diff
@@ -1284,14 +1284,14 @@ I will never do anything that^will make Arcueid cry ever again.^Even if my brain
 ロアの死体が焼かれていく。^街を脅かす脅威は去った。^吸血鬼事件は終わったのだ。
 これで終わり。^明日からは平穏に生きられる。^傷や思い出も、全部忘れて――
 クリスマスのプレゼント。^代行者としての責務と、^復讐者としての情熱と。
-'It's best if we don't see each^other anymore.' Arcueid said^that. As a human, that would^definitely be the right choice.
+Arcueid said 'It's best if we don't see each^other anymore.'^As a human, that would^definitely be the right choice.
 It's after-school. The rest of^the day is mine. Then, what I^should do now is――
-I should pretend none of this^ever happened. I should think^of all of this as a dream. Is^that really―――really it?
+I should pretend none of this^ever happened like all of this^was a just dream. Is that really―――really it?
```
Comment (rschlaikjer) [inline system_strings/sysmes_text.en:1289]: Needs some sort of punctuation after `happened`. Either a period or some em dashes

File: script/Ciel/Day 9/09_01_CIEL09_2B.txt [hunk 13]
```diff
@@ -98,7 +98,7 @@ By telling me more about Arcueid and about vampires, she's giving exactly what I
 [sha:18bcc86b34664b8759253cc7931477116a07ccd1]{
 -- Page 156, Offset 22528. Glued.
 -- ホモサピエンスの血液を糧とする異物を、わたしたち聖堂教会は吸血鬼と呼称しています」
-We of the Church refers to those who suck the blood of primates and of homo sapiens as nourishment as 'vampires'."
+We of the Church refers to those who suck the blood of hominoid and of homo sapiens as nourishment as 'vampires'."
```
Comment (yk1512) [inline script/Ciel/Day 9/09_01_CIEL09_2B.txt:101]: hominoids

**Review by yk1512 — APPROVED** [review]

File: script/Arcueid/Day 10/10_00_ARC10_6.txt [hunk 14]
```diff
@@ -1033,6 +1033,7 @@ In common terms, those people are referred to as %{i}Psychics%{/i}, or %{i}Chang
 [sha:962b172f1da03cd553a7228367bde0539d3a665c]{
 -- Page 732, Offset 26262.
 -- 特異能力ってのは肉体的なものだからな、家系―――血筋で遺伝する。ロアは自分の新しい肉体にそういった『人間ではないもの』の家系を選ぶって話だ」
+// dont change the abilities here?
```
Comment (Avuxo) [inline script/Arcueid/Day 10/10_00_ARC10_6.txt:1036]: yes this one is good as-is

File: script/Arcueid/Day 8/08_00_ARC08_8.txt [hunk 15]
```diff
@@ -359,6 +359,7 @@ The impact of knife clashing against knife.
 -- Page 504, Offset 21019.
 -- 理性を超越した本能が刃を弾く。
 // 理性 intentionally omitted, feels clear from phrase.
+// group pr check: i missed what's supposed to be the change here
```
Comment (Avuxo) [inline script/Arcueid/Day 8/08_00_ARC08_8.txt:362]: no change here, we were just talking about how 超越 is translated

File: system_strings/sysmes_text.en (hunk 5 above)
Comment (Avuxo) [reply system_strings/sysmes_text.en:1120]: typo i think

File: system_strings/sysmes_text.en (hunk 10 above)
Comment (Avuxo) [reply system_strings/sysmes_text.en:1253]: yes it does matter, typo

File: system_strings/sysmes_text.en (hunk 11 above)
Comment (Avuxo) [reply system_strings/sysmes_text.en:1255]: just a typo. should be "I believe"

File: system_strings/sysmes_text.en (hunk 4 above)
Comment (Avuxo) [reply system_strings/sysmes_text.en:644]: the problem was "focus on lesson". No space to add "the" so we had to reword the whole line

**Review by Avuxo — APPROVED** [review]

**Comment by Avuxo** [comment]
Wait for PR 710 before merging this down

File: system_strings/sysmes_text.en (hunk 5 above)
Comment (Volvo-hackangel) [reply system_strings/sysmes_text.en:1120]: it is, my bad

Comment (Avuxo) [reply system_strings/sysmes_text.en:1120]: can't exactly let you take the blame for things that i wrote :aokoheh:.

File: script/Arcueid/Day 4/04_00_ARC04_6_2B.txt [hunk 16]
```diff
@@ -407,7 +407,7 @@ Wouldn't it be easier to just do it all by yourself without creating these serva
 [sha:e435916eca9d3de99b264c5b381753bc5ef102e4]{
 -- Page 501, Offset 6940.
 -- どれほどの超抜能力、どれほどの<存在規模|ライフスケール>をもった吸血鬼でも、<単身|ひとり>では生きられない。
-No matter how much supernatural power they have, or how large their scope of existence is, vampires cannot survive on their own.
+No matter how much superpower they have, or how large their Lifescale is, vampires cannot survive on their own.
```
Comment (rschlaikjer) [inline script/Arcueid/Day 4/04_00_ARC04_6_2B.txt:410]: Seems a bit weird to have a _quantity_ of a superpower? I feel like they're usually referred to in terms of extent, potency?
> No matter how incredible their superpowers may be, or ...
> No matter how powerful they are, or ...

Comment (yk1512) [reply script/Arcueid/Day 4/04_00_ARC04_6_2B.txt:410]: I'm fine with the former

**Pull request merged**